### ISSUE TITLE
Failed test

### ISSUE THREAD

**Issue by kevinresol** [haxetink/tink_sql#57]
Was trying to implement #38 while I found a strange problem, if I change the value to `6` at [this line](https://github.com/haxetink/tink_sql/blob/5240624/src/tink/sql/drivers/node/MySql.hx#L31) then the diffEmpty test fails, any idea? @benmerckx

**Comment by benmerckx** [comment]
How does it fail? The schema tests fail sometimes at random but I've not been able to figure out why exactly. My best and probably worst guess at this time is the schema data is cached and sometimes stale data is returned by mysql.

**Comment by benmerckx** [comment]
I'm seeing the same:
````
  diffEmpty: [SchemaTest.hx:33]
    - [OK] [SchemaTest.hx:35] changes.length == 27 (27 == 27)
    - [FAIL] [SchemaTest.hx:23] diff.length == 0 (1 == 0)
`````
I'll look into it

**Comment by benmerckx** [comment]
The order of the columns returned by mysql and used in `Schema.fromMysql` is wrongly assumed to be the same order as the table declation. This messes up the index field order as well which produces a ChangeIndex diff. I can fix this when I work in #47 (at least the 'data types' part).

**Comment by benmerckx** [comment]
Should be okay with ab4df6a until I can look into fixing this properly (separate index data from column info completely).
